odl/trafos/wavelet.py

```diff
@@ -438,17 +467,42 @@ def __init__(self, range, wavelet, nlevels=None, pad_mode='constant',
         """
         super(WaveletTransformInverse, self).__init__(
             space=range, wavelet=wavelet, variant='inverse', nlevels=nlevels,
-            pad_mode=pad_mode, pad_const=pad_const, impl=impl)
+            pad_mode=pad_mode, pad_const=pad_const, impl=impl, axes=axes,
+            slices=slices, shapes=shapes)
 
     def _call(self, coeffs):
         """Return the inverse wavelet transform of ``coeffs``."""
+        if self._coeff_slices is None:
+            raise ValueError(
+                "self._coeff_slices not initialized.  Run the forward "
```

Comment:
I agree this is not ideal (however, in practice I think it is unlikely one would start with coefficients generated elsewhere and apply the inverse transform without having first done the forward transform). 

A simple way to fix this would be to just run a dummy transform during  `__init__`, but that has some computational overhead. Another alternative is to make a small utility function that can generate the appropriate slices without actually running the transform. I don't think there is a functio n for that in PyWavelets, but it shouldn't be too hard to implement if you prefer that route.